src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -7,15 +7,75 @@ import { SummaryService } from '../../../shared/services/summary.service';
   styleUrls: ['./navigation.component.scss']
 })
 export class NavigationComponent implements OnInit {
+  user: string;
+  browser: string;
+  browserVersion: string;
+  browserOS: string;
   summaryData: any;
   rbdPools: Array<any> = [];
 
-  constructor(private summaryService: SummaryService) {}
+  constructor(private summaryService: SummaryService) { }
 
   ngOnInit() {
     this.summaryService.summaryData$.subscribe((data: any) => {
       this.summaryData = data;
       this.rbdPools = data.rbd_pools;
+      this.user = localStorage.getItem('dashboard_username');
+      this.browser = navigator.appName;
```

Comment:
Great addition! Could we use some NPM package (JS or ng2 specific), like [this one](https://www.npmjs.com/package/detect-browser)? The idea is to reuse packages as much as possible.